### DISCUSSION TITLE
How to redeem non-profit GitHub Teams discount? No response from GitHub Support.

### DISCUSSION THREAD

**Discussion by kyleecodes** [community/community#114311] [New to GitHub]
Greetings, I work with a non-profit and we applied for our non-profit Teams discount months ago.
See ticket here: https://support.github.com/ticket/personal/0/2439316 
The above ticket was archived before I could respond. It says we were already approved and Teams was redeemed in October, but Teams has never been redeemed (see pics below). There is also no option for a discount showing up when upgrading.
I opened another ticket to ask how this can be redeemed, but we're going on almost 2 months now with no answer: https://support.github.com/ticket/personal/0/2581729#tc-23516148876948
Also, according to [these docs](https://docs.github.com/en/billing/managing-the-plan-for-your-github-account/discounted-plans-for-github-accounts#discounts-for-nonprofits-and-libraries), if we pay for the subscription first we will be refunded. But this doesn't clarify how exactly the discount works. As a non-profit, we need to be very careful with our budgeting, so we don't want to pay for the subscription unless we 100% know we will be refunded and when. 

Can we get help sorting this out? Thank you. 

Our subscription is the Free plan:
![Capture](https://github.com/community/community/assets/43586156/0870f7cd-3cbb-4c9b-9fb6-88ccca8c988e)
We have never paid for teams:
![Capture1](https://github.com/community/community/assets/43586156/eaa27629-6707-46a2-87fc-9b172720def8)
No non-profit discount shown:
![capture3](https://github.com/community/community/assets/43586156/a3919926-6740-43ad-a9cb-da5b266d3d27)

**Comment by queenofcorgis** [comment]
Hey @kyleecodes 👋🏼 

Good to see you again especially in the Community! That's awesome you're working with Chayn - I'm a big fan of their work 💟 . 

I appreciate all your patience here. The community is unable to assist with nor escalate your ticket, but I did check on it and we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
